src/components/CodeSnippet/CodeBlock.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import copyCode from "./CopyCode";
+import customTheme from "./CustomTheme";
+import { CodeBlock} from 'react-code-blocks';
+import styles from './codeBlock.module.scss';
+import { copyIcon } from "./CopyIcon";
+
+//Documentation for react-code-blocks: https://www.npmjs.com/package/react-code-blocks
+
+interface CodeBlockProps {
+    language: string;
+    code: string;
+
+  }
+
+export function CodeSample(props: CodeBlockProps): JSX.Element{
```

Comment:
The component should match the file name. So if we're going with CodeBlock.tsx, it needs to return CodeBlock().

Reply:
Makes sense. Since "CodeBlock" is a reserved keyword from the library "'react-code-blocks" that I am using I decided to rename the filename to CodeSample.tsx